### PR TITLE
[ethermon-erc721] ethermon-erc721

### PR DESCRIPTION
**Pull request by syedMohib44**
Fixes # .

Initial commit.

### CODE REVIEW

**Comment by syedMohib44** [comment]
[Version 0.0.1] ethermon-erc721

File: src/strategies/ethermon-erc721/index.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { Multicaller } from '../../utils';
+
+export const author = 'mariana44';
```
Comment (ChaituVR) [inline src/strategies/ethermon-erc721/index.ts:4]:
```suggestion
export const author = 'syedMohib44';
```

File: src/strategies/ethermon-erc721/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# ethermon-erc721
```
Comment (ChaituVR) [inline src/strategies/ethermon-erc721/README.md:1]: Please revert changes from the package-lock file

File: test/index.spec.ts [hunk 3]
```diff
@@ -159,11 +158,10 @@ describe(`\nTest strategy "${strategy}" with latest snapshot`, () => {
       await new Promise((r) => setTimeout(r, 500));
     });
 
-    it(`Should take less than 20 sec. to resolve with ${
-      moreArg || 500
-    } addresses`, () => {
-      expect(getScoresTimeMore).toBeLessThanOrEqual(20000);
-    });
+    it(`Should take less than 20 sec. to resolve with ${moreArg || 500
```
Comment (ChaituVR) [inline test/index.spec.ts:161]: Please revert changes from this spec file

File: test/delegation.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ const snapshot = require('../').default;
 
 const space = 'cvx.eth';
 const network = '1';
-const addresses = ['0xde1E6A7ED0ad3F61D531a8a78E83CcDdbd6E0c49'];
+const addresses = ['0x038ae33f4bbfd9e0489abd7e622f014244433f72'];
```
Comment (ChaituVR) [inline test/delegation.ts:5]: Also revert this plz

File: src/strategies/ethermon-erc721/examples.json [hunk 5]
```diff
@@ -0,0 +1,1484 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "ethermon-erc721",
+      "params": {
+        "EMON_DATA_ADDRESS": "0x5d00d312e171be5342067c09bae883f9bcb2003b",
+        "EMONA_ADDRESS": "0xabc1c404424bdf24c19a5cc5ef8f47781d18eb3e",
+        "symbol": "EMONA",
+        "classIdWeight": {
```
Comment (ChaituVR) [inline src/strategies/ethermon-erc721/examples.json:10]: It is not good idea to store all these weights inside strategy params, maybe you can store these in a IPFS file and fetch it when required?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by syedMohib44 — COMMENTED** [review]
Changes resolved

**Review by syedMohib44 — COMMENTED** [review]
Resolved name changed

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@syedMohib44 Still see changes in package-lock and also need to move weights in examples.json to IPFS

**Comment by syedMohib44** [comment]
> @syedMohib44 Still see changes in package-lock and also need to move weights in examples.json to IPFS

Alright @ChaituVR on it

**Comment by syedMohib44** [comment]
@ChaituVR can you verify the changes once. Thanks

**Comment by syedMohib44** [comment]
> Hey @syedMohib44 Thanks for the changes, looks good , except that import `ethermon-erc721` strategy into src/strategies/index.ts like other strategies, Thanks

Hii @ChaituVR imported the ethermon-erc721 stragtegy in index.ts can please review it once. Thanks you.

**Comment by syedMohib44** [comment]
> @syedMohib44 IPFS file doesn't resolve well, Is it pinned correctly? make sure to use a third-party provider like pinata or fleek
> 
> Or is there any other way to get these weights on chain? maybe from subgraph?

@ChaituVR there is a problem with IPFS I guess 😄 previously it was pinned and not working that 504 gateway problem then I unpinned it and worked not sure what was the issue but ipfs is not reliable can do test now and let us know. Will update this to some reliable solution in next version.

**Comment by syedMohib44** [comment]
> Hey @syedMohib44 Thanks for the changes, looks good , except that import `ethermon-erc721` strategy into src/strategies/index.ts like other strategies, Thanks

@ChaituVR all looks good please view import of ethermon-erc721 once would be awesome.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**